src/main/resources/config.sk

```diff
@@ -186,6 +186,16 @@ keep command last usage dates: false
 # but when the player uses the command again after the cooldown period is over, the last usage will be deleted as it's no longer needed,
 # If you need to use the expression 'last usage date', then you'll want to enable this.
 
+script loader thread size: 0
```

Comment:
Might be useful to accept a value like `1t` similar to Maven's command line argument `threads`, which will use one thread per CPU core, so it can utilize the maximum power of the processor (unless those threads are blocked on I/O, memory, etc. In that situation having more threads than CPU cores may improve performance, otherwise it will just lower down performance because of context switching).